src/main/java/cloud/eppo/api/Configuration.java

```diff
@@ -98,20 +92,6 @@ public interface Callback {
     this.bandits = bandits;
     this.isConfigObfuscated = isConfigObfuscated;
 
-    // Graft the `forServer` boolean into the flagConfigJson'
-    if (flagConfigJson != null && flagConfigJson.length != 0) {
```

Comment:
no longer needed no that the `FORMAT` field exists in the UFC

Reply:
🚀